artifacts/checksums.txt

```diff
@@ -1,10 +1,13 @@
-c5a2673b74cda51bd2628804856117291ed555867daf66f543e084dccc694576  cw4_voting.wasm
+e1f198ec554470bd05f4081cf2309035a1369152b147bdc2ca2fad7120364641  cw4_group.wasm
```

Comment:
does it belong here? I'd move it to the integration tests repo instead

Reply:
I have no idea, the only thing I have done here is run `make build`

Reply:
sorry, I meant `cw4_group.wasm`. it shouldn't be a result of `make build`. it's not even in the `checksum.txt` file

Reply:
because it's required for the test script